docs/user-guide/nginx-configuration/annotations.md

```diff
@@ -213,6 +214,15 @@ The ingress controller requires a [default backend](../default-backend.md).
 This service handles the response when the service in the Ingress rule does not have endpoints.
 This is a global configuration for the ingress controller. In some cases could be required to return a custom content or format. In this scenario we can use the annotation `nginx.ingress.kubernetes.io/default-backend: <svc name>` to specify a custom default backend.
 
+### Custom HTTP Errors
+
+Like the ConfigMap `custom-http-errors`, this annotation will set NGINX `proxy-intercept-errors`, but only for the hostname associated with this ingress. Different ingresses can specify different sets of error codes. If multiple ingress objects share the same hostname, this annotation can be used to intercept different error codes for each hostname. If `custom-http-errors` is also specified globally, the error values specified in this annotation will override the global value for this ingress' hostname and path.
```

Comment:
Not sure I follow this sentence: `If multiple ingress objects share the same hostname, this annotation can be used to intercept different error codes for each hostname. `. Maybe you can explain in different words?

Reply:
ah typo - should say `can be used to intercept different error codes for each path`.

Effectively, if there are two different ingresses for the same host but different paths, each ingress can specify its own set of `custom-http-errors` (which then applies for the corresponding path of that ingress).

eg:
An ingress object for api.company.com/v1 can override "500,502,503", and a different ingress object for api.company.com/v2 can override "401,503,504" etc, and that will be respected.